app/views/shared/_page_header.html.erb

```diff
@@ -7,11 +7,11 @@
     <%= switch_service_button if show_switch_service %>
   </span>
 
+  <%= extra_content %>
```

Comment:
This is significantly changing the behaviour of the `_page_header` partial which will affect the display of other pages using `extra_content`

The  partial is already quite flexible, and if it is insufficiently flexible then its better to either not use it in the header page or failing that improve its flexibility further.

Reply:
yes, as discussed we actually do not need to move the extra_content 👍